components/core/src/clp/TimestampPattern.cpp

```diff
@@ -182,6 +182,8 @@ void TimestampPattern::init() {
     patterns.emplace_back(4, "%a %b %e %H:%M:%S %Y");
     // E.g. Sun Jan  1 15:50:45 2015
     patterns.emplace_back(0, "%a %b %e %H:%M:%S %Y");
+    // E.g. +0800 1970-01-01 08:00:00.235
+    patterns.emplace_back(0, "%z %Y-%m-%d %H:%M:%S.%3");
```

Comment:
* `%z` is not a supported format specifier. You're probably thinking of a [change](https://github.com/kirkrodrigues/clp/blob/timezone-support/components/core/src/clp/TimestampPattern.hpp#L49) that hasn't yet been fully merged. @LinZhihao-723 was working on it but he has other priorities at present.
* We should also add a unit test for any new timestamp pattern we add.